### ISSUE TITLE
Module `@react-native-community/async-storage` does not exist in the Haste module map

### ISSUE THREAD

**Issue by zashishz** [moaazsidat/react-native-qrcode-scanner#198]
Unable to use react-native-qrcode-scanner on react-native v-0.60+
Here is my package.json file
```json
"dependencies": {
    "axios": "^0.19.0",
    "prop-types": "^15.7.2",
    "react": "16.8.6",
    "react-native": "0.60.4",
    "react-native-camera": "^3.1.1",
    "react-native-gesture-handler": "^1.3.0",
    "react-native-linear-gradient": "^2.5.6",
    "react-native-qrcode-scanner": "^1.2.1",
    "react-navigation": "^3.11.1",
    "react-redux": "^7.1.0",
    "redux": "^4.0.4",
    "redux-persist": "^5.10.0",
    "redux-saga": "^1.0.5"
  },
```

### What's happening?
I cannot use react-native-qrcode-scanner on react-native v-0.60+.
tried manually linking too. but nothing helped.

### How can it be reproduced?
create a new react-native project v0.60+
yarn add react-native-camera
yarn add react-native-qrcode-scanner

follow setup instructions mentioned [here](https://github.com/moaazsidat/react-native-qrcode-scanner#getting-started).

Use the QRCodeScanner component.
```js
import React from 'react';
import { View, Text } from 'react-native';
import PropTypes from 'prop-types';
import QRCodeScanner from 'react-native-qrcode-scanner';

const Scan = ({ navigation }) => (
  <>
      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
        <Text>
            Scan
        </Text>
        <QRCodeScanner
          onRead={() => 'Hi'}
        />
      </View>
  </>
);

Scan.propTypes = {
  navigation: PropTypes.object,
};

export default Scan;

```

### Build details?
Can be reproduced on android and iOS as well.

**Comment by zashishz** [comment]
For now i have manually linked 
```
@react-native-community/async-storage
react-native-camera
react-native-permissions
```
did pod install for iOS and also added below line into my app build.gradle file
```
defaultConfig {
    ...
    missingDimensionStrategy 'react-native-camera', 'general'
}
```

Thanks to @manelsanz  for [this](https://github.com/react-native-community/react-native-camera/issues/2150#issuecomment-484059264)

**Comment by lucasvdev** [comment]
@zashishz Hello,

Did you just execute theses lines ? 
```
react-native link react-native-camera
react-native link @react-native-community/async-storage
react-native link react-native-permissions
```

I'm facing this same issue, I mannualy linking dependencies like as above but I still got the error...

**Comment by regalfaith** [comment]
@lucasvdev This is how I solved the problem on RN 0.60.5 and iOS13

The problem was that there is a breaking change in react-native-permissions 1.2.1 relating to '@react-native-community/async-storage'

And the reason why this issue happened was because the correct version(1.1.1) of react-native-permissions was not installed alongside when react-native-qrcode-scanner was installed.
Instead, the version 1.2.1 kept being installed.

So here's what I did to solve this issue. **And this is only specific to RN >= 0.60**

---
1. add react-native-permissions dependency in **package.json**
```
    "react-native-permissions": "^1.1.1",
```

2. add react-native-permissions dependency **package-lock.json** to force version 1.1.1
```
    "react-native-permissions": {
      "version": "1.1.1",
      "resolved": "https://registry.npmjs.org/react-native-permissions/-/react-native-permissions-1.1.1.tgz",
      "integrity": "sha512-t0Ujm177bagjUOSzhpmkSz+LqFW04HnY9TeZFavDCmV521fQvFz82aD+POXqWsAdsJVOK3umJYBNNqCjC3g0hQ=="
    },
```

3. Do **Not** react-native link both react-native-qrcodescanner and react-native-permissions but just **pod install**(if needed do linking manually in Android)
```console
    cd ./ios && pod install && cd ..
```

---
